packages/steiger-plugin-fsd/src/_lib/prepare-test.ts

```diff
@@ -4,8 +4,31 @@ import type { FsdRoot } from '@feature-sliced/filesystem'
 import type { Folder, File, Diagnostic } from '@steiger/types'
 import { vi } from 'vitest'
 
+function findSubfolder(folder: Folder, path: string): Folder {
+  function checkFolder(folder: Folder): Folder {
+    if (folder.path === path) {
+      return folder
+    }
+
+    if (path.startsWith(folder.path)) {
+      for (const child of folder.children) {
+        if (child.type === 'folder') {
+          const result = checkFolder(child)
+          if (result) {
+            return result
+          }
+        }
+      }
+    }
+
+    throw new Error(`Path "${path}" not found in the provided file system mock!`)
+  }
+
+  return checkFolder(folder)
+}
```

Comment:
suggestion: I meant a more simple solution — you pass the root path as the second argument, and whatever file tree you write with emojis is "mounted" to that root. For example:

```ts
parseIntoFsdRoot(`
  📂 test
    📄 file
`, "/home/illright");  // the "file" becomes `/home/illright/test/file`
```

Reply:
Yeah, I agree, the simpler the better, the [KISS](https://www.youtube.com/watch?v=ZhIsAZO5gl0) principle should not be forgotten 😂